common/lib/xmodule/xmodule/js/js_test.yml

```diff
@@ -59,12 +59,18 @@ lib_paths:
     - common_static/js/src/utility.js
     - public/js/split_test_staff.js
     - common_static/js/src/accessibility_tools.js
+    - common_static/js/vendor/moment.min.js
 
 # Paths to spec (test) JavaScript files
 spec_paths:
     - spec/helper.js
     - spec
+    - spec/main_requirejs.js
 
 # Paths to fixture files (optional)
 fixture_paths:
     - fixtures
+
+requirejs:
```

Comment:
I was trying different alternatives; in order to make the moment.js workable for specs. i should remove these changes as i'm not using these anymore. Thanks for pointing this out.